contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/util/SimpleHttp.java

```diff
@@ -870,6 +888,22 @@ public static HttpStoragePlugin getStoragePlugin(DrillbitContext context, String
     }
   }
 
+  public static String addBackTicksToAliasName(String plugin, String identifier) {
+    plugin = plugin.trim();
+    if (! plugin.startsWith(identifier)) {
+      plugin = identifier + plugin;
+    }
+    if (! plugin.endsWith(identifier)) {
+      plugin = plugin + identifier;
+    }
+    return plugin;
+  }
+
+  public static String removeBackTicksFromPluginName(String plugin, String identifier) {
+    plugin = plugin.trim();
+    plugin = plugin.replaceAll(identifier, "");
+    return plugin;
+  }
```

Comment:
Looks like these two methods aren't used and should be removed.

Reply:
Removed.